vivisect/parsers/elf.py

```diff
@@ -13,6 +13,10 @@
 
 logger = logging.getLogger(__name__)
 
+
+def isParser(bytez):
+    return bytez.startswith("\x7fELF")
```

Comment:
```suggestion
    return bytez.startswith(b"\x7fELF")
```

Reply:
i haven't tested this, but seems like the `bytez` should be bytes, in which case we can't provide strings here. or am i wrong?

Reply:
in this case, it's actually `bytez`, as it's returning a bool of whether the bytez bytes starts with `\x7fELF`

Reply:
sorry, this just hit me what you really meant.